.examples/docker-compose-postgres-storage/docker-compose.yml

```diff
@@ -18,6 +18,10 @@ services:
     restart: always
     ports:
       - "8080:8080"
+    environment:
+      - POSTGRES_DB=gatus
+      - POSTGRES_USER=username
+      - POSTGRES_PASSWORD=password
```

Comment:
```suggestion
      - POSTGRES_USER=username
      - POSTGRES_PASSWORD=password
      - POSTGRES_DB=gatus
```

Reply:
Nitpicking, but just to match the order of the env vars as they're used in `postgres://${POSTGRES_USER}:${POSTGRES_PASSWORD}@postgres:5432/${POSTGRES_DB}?sslmode=disable`

Reply:
I matched the order from the postgres container in this file. Do you want postgres changing too or leaving as it is?

Reply:
No need, just changing the order of the env vars as I suggested will be enough.
I'd have committed it directly in your PR, but I think you disabled the option allowing contributors to contribute to your PR/fork